sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ExpressionEvalHelper.scala

```diff
@@ -57,8 +57,8 @@ trait ExpressionEvalHelper extends GeneratorDrivenPropertyChecks {
     (result, expected) match {
       case (result: Array[Byte], expected: Array[Byte]) =>
         java.util.Arrays.equals(result, expected)
-      case (result: Double, expected: Spread[Double]) =>
-        expected.isWithin(result)
+      case (result: Double, expected: Spread[_]) => // Can't use Spread[Double] b/c of erasure
```

Comment:
maybe I am not familiar with the style guidelines on this point, but it seems to be a perfect use case for `@unchecked`: http://www.scala-lang.org/api/current/index.html#scala.unchecked